config/vufind/Search2.ini

```
@@ -307,3 +307,17 @@ facet_limit      = 20
 
 [HideFacetValue]
 ;format[] = "Book"
+
+; Settings specific to a facet field. Each section is named [Facet_<field_name>].
+; The example section is used for publishDate. Additional sections can be added for
+; other fields as needed.
+[Facet_publishDate]
+; Minimum value for date range sliders in search forms (default is 1400). Value
+; must be a number or a string that PHP's strtotime can parse. Only the year part
+; is used.
+;slider_min_value = "1400"
+
+; Maximum value for date range sliders in search forms (default is current
+; year + 1). Value must be a number or a string that PHP's strtotime can parse.
+; Only the year part is used.
+;slider_max_value = "+1 year"
```

Removing comments here for consistency with the rest of the file (there's a "see other files for comments" note at the top):

```suggestion
[Facet_publishDate]
;slider_min_value = "1400"
;slider_max_value = "+1 year"
```